Library/src/Common/String.cpp

```diff
@@ -39,15 +39,22 @@ class String::Impl
 
 	explicit Impl(char const* const InText) : Text(nullptr), Length(0)
 	{
+		const char* _InText = InText;
+
 		if (InText == nullptr)
 		{
-			return;
+			_InText = "";
 		}
 
-		const size_t Len = strlen(InText);
+		const size_t Len = strlen(_InText);
```

Comment:
Out of interest, what did this fix?

Reply:
Text can't be null, so we need to set it to something.